doc/src/manual/strings.md

```diff
@@ -516,9 +514,8 @@ false
 julia> contains("Xylophon", 'o')
 ERROR: MethodError: no method matching contains(::String, ::Char)
 Closest candidates are:
-  contains(!Matched::Function, ::Any, !Matched::Any) at reduce.jl:640
-  contains(::AbstractString, !Matched::AbstractString) at strings/search.jl:366
- ...
+  contains(::Function, ::Any, ::Any) at reduce.jl:628
```

Comment:
Doctests are currently run in the same process. That's probably not too much trouble to change though.

Regarding the color output here though: we can just switch off color output during doctest printing in Documenter with something like `eval(Base, :(have_color = false))`, but might be preferable to add an official way to do that in `Base` rather than hacking it like that.